README.md

```diff
@@ -7,1334 +7,81 @@
-configure the CLI to work with local endpoints so you can interact with the
-database running in the Docker container.
-
-The docker container is explained in depth here:
-https://docs.fauna.com/fauna/current/tools/dev.
-
-Once you've installed the Shell and logged in, you can configure it by doing the
-following:
-
-By default, the Fauna Docker image serves data via port 8443. To add a
-connection to this port, run `fauna endpoint add`. The Database URL should be
-`http://localhost:8443` (the default), and the database secret should be
-`secret` by default.
-
-```sh
-$ fauna endpoint add
-? Endpoint name localhost
-? Database URL http://localhost:8443
-? Database Secret secret
-Checking secret... done
-? Make this endpoint default [no]
-Saved endpoint localhost to ~/.fauna-shell
-```
-
-Now, you can interact with your local database through the Fauna Shell by
-running the command below:
-
-```sh
-fauna shell --endpoint localhost
+npm update -g fauna-shell
 ```
 
-# Overriding Connection Parameters
-
-Most commands support the following options. You can specify them if you want to
-connect to a local instance of Fauna.
+## Documentation
 
-```sh
-OPTIONS
-  --endpoint=name      Selects an endpoint from ~/.fauna-shell
-  --endpointURL=domain Overrides the `url` setting in the selected endpoint.
-  --secret=secret      Overrides the `secret` setting in the selected endpoint.
-  --timeout=timeout    [default: 5000] Connection timeout in milliseconds
-```
+For usage instructions and a full [list of
+commands](https://docs.fauna.com/fauna/current/build/cli/commands/), see the
+[Fauna docs](https://docs.fauna.com/fauna/current/build/cli/).
 
-`--endpoint` doesn't need to be set if `--endpointURL` and `--secret` are provided.
-
-They can be used like this:
+You can also access help directly in the CLI by running:
 
 ```sh
-$ fauna create-database testdb --endpointURL=http://127.0.0.1:8443 --secret=YOUR_FAUNA_SECRET_KEY --timeout=42
-```
-
-You could select another endpoint from `~/.fauna-shell` using `--endpoint`:
-
-```sh-session
-$ fauna shell --endpoint=localhost
-```
-
-Any options that are not specified either via the `.fauna-shell` config file or
-the CLI will be set to the defaults offered by the
-[JS driver](https://github.com/fauna/fauna-js).
-
-# Executing queries from a file
-
-You can also tell the shell to execute a list of queries that you have stored in
-a file. For example, you can have a file that creates a collection called
-`setup.fql`:
-
-```ts
-Collection.create({
-  name: "Post",
-  indexes: {
-    byTitle: {
-      terms: [{ field: ".title" }]
-    }
-  }
-})
-```
-
-Once the collection is created, you can execute queries against it in another
-`.fql` file:
-
-```ts
-Post.create({
-  title: "What I had for breakfast .."
-})
-
-[
-  "My cat and other marvels",
-  "Pondering during a commute",
-  "Deep meanings in a latte",
-].map(title => {
-  Post.create({
-    title: title
-  })
-})
+fauna --help
```

Comment:
`fauna --help` or `fauna [subcommand] --help` may make it clearer that there's additional help on each subcommand?

Reply:
Good Idea. Added with [`f01bb7e` (#396)](https://github.com/fauna/fauna-shell/pull/396/commits/f01bb7e1fa5025c0babe8553956826c8260dec5f)